### DISCUSSION TITLE
Getting more people involved

### DISCUSSION THREAD

**Discussion by djmitche** [taskchampion/taskchampion#255] [General]
So far, it's pretty much been me and @dbr hacking on TC.  How can we get more people involved?

 * Are there people who would appreciate doing some basic Rust work as a way to practice?
 * Can we get the app useful enough that it will "hook" people who have the capacity to then scratch their own itches?  Without spending thousands of hours finishing it first?
 * Are there funding models where we could compensate people with the skills to work on it -- bounties? contracts?
 * Other ideas?

**Comment by dbr** [comment]
Not sure how effective it is, but the "This Week in Rust" project has a "Call for Participation" section, e.g https://this-week-in-rust.org/blog/2021/05/12/this-week-in-rust-390/

..but I think the most important thing is as you say, getting TC in a state where we can reasonably promote it as "WIP but usable" (which is probably not much beyond the 0.4.0 milestone). Since it's a CLI based application it probably naturally aims more towards users who are programmers, and the barrier to contribution should be fairly low as it is a fairly simple application

**Reply by djmitche** [reply]
Good thinking about TWiR!

Yeah, compared to other stuff I've worked on this is definitely "not-too-complicated Rust".  Not performance-sensitive, mostly owned data, single-threaded, not async, ..
